src/lib/sections/scan-an-nft.svelte

```diff
@@ -61,6 +63,11 @@
 		let nftContract: Contract = new ethers.Contract(nftContractAddress, defaultAbi, $user.provider);
 		let tokenUri = await nftContract.tokenURI(tokenId);
 		metaData = await tokenUriToMetaData(tokenUri);
+
+		evaluation = await evaluateNft(metaData);
```

Comment:
This function wants the `tokenUri` directly, not the metadata. It needs to see what the actual tokenUri contains (if it is a link, etc).

Reply:
Gotcha! Will fix rolling forward. Thanks